### DISCUSSION TITLE
Notes for October 17th meeting

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#102] [General]
### Update on issues

* Issue #99: created, to discuss and assign

* Issue #89: closed

* Issue #84: tagged for version 2

* Issue #81: closed, split into issues #97 and #99

* Issue #78: Turtle finalized and text added to the specification, including Zoltán's example; issue closed with commit https://github.com/linked-statistics/COOS/commit/fc58246760bdc2b998f777d60c88efa5c458e791

* Issue #76: deadline for NTTS passed

* Issue #72: nothing new

* Issue #69: nothing new

* Issue #65: comments by InKyung, to discuss

* Issue #63: nothing new

* Issue #60: InKyung and Flavio contributed, to discuss

* Issue #59: pull request #103 created for review by Jenny

* Issue #19: tagged for version 2

### Ontology expert feedback

* Documenting the source standard for the different items ([ref](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#documentation)): issue to create and tag for version 2.

* Application profile ([ref.](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#application-profile)): issue to create and tag for version 2.

* Activity/Activity category ([ref.](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#activity--statisticalactivity)): pull requests created by Florian for [Turtle file](https://github.com/linked-statistics/COOS/pull/100) and [HTML specification](https://github.com/linked-statistics/COOS/pull/101), reviewed by Flavio and Thomas, and merged

### Public review

* Various improvement on document (https://github.com/linked-statistics/COOS/commit/54d70b14a939ac0910204f5c61d5e7029a2f3375), help needed for summary.

* Discuss practical organization. See comments made by InKyung and Flavio on discussion #96.

### AOB

**Comment by InKyungChoi** [comment]
**Meeting notes (October 17th)**

Participants: Franck Cotton, Jenny Linnerud, Flavio Rizzolo, Zoltan Vereczkei, Juan Munoz, Christine Laaboudi, Florian Vucko, Paolo Francescangeli and InKyung Choi

- Invitation to public review to be sent out to those in [this list](https://github.com/linked-statistics/COOS/discussions/96#discussioncomment-3896494)
- #99 In the new GSIM, the relationship between Product and Information Set is "has" while it is a sub-type relationship in COOS. It is okay to have both relationships, leave this issue open for now. 
-  #78 The examples of reference document from Zoltan were added to COOS specification. Issue to be closed. 
-  #76 Zoltan reported that discussion with NTTS organisers is on-going, there might be a session on ModernStats models which will include COOS
- #65 InKyung proposed to add "Other National Organisation". "Other National Authority" had been originally proposed but not included in the end because this has specifical meaning in ESS, but "Other National Organisation" might work - to keep open to discuss
- #60 The activities under “overarching activities” (e.g., overarching process, GSBPM phase 1-3+8) are different from the usual production phases (GSBPM phase 4-7), it would be good to have a separate class to represent them, but the name is still to be decided. GSBPM and GAMSO revision is coming up, we should try to see if there is a way to represent them in ontology that is not too sensitive to changes of underlying models
